src/Protocol/Feed.php

```diff
@@ -945,7 +945,7 @@ public static function atom(array $owner, string $last_update, int $max_items =
 
 		$previous_created = $last_update;
 
-		// Don't cache when the last item was posted less then 15 minutes ago (Cache duration)
+		// Don't cache when the last item was posted less than 15 minutes ago (Cache duration)
```

Comment:
this isn't technically a spelling error, but even very old spell checkers flag such a thing, so check-spelling does as well